docroot/modules/custom/mass_entity_usage/src/MassEntityUsage.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Drupal\mass_entity_usage;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\entity_usage\EntityUsage;
+
+/**
+ * Extends the entity usage base class.
+ */
+class MassEntityUsage extends EntityUsage implements MassEntityUsageInterface {
+
+  /**
+   * {@inheritdoc}
```

Comment:
Instead of inheritdoc, how about we say why we are overriding the parent method.

Reply:
Ended up not needing this method